client/js/templates/Item.jsx

```diff
@@ -444,7 +487,7 @@ export default function Item({ currentTime, item, selected, expanded, setNavExpa
             {/* title */}
             <h3
                 className="entry-title"
-                onClick={titleOnClick}
+                onClick={configuration.doubleClickMarkAsRead ? titleOnMultiClick : titleOnClick}
```

Comment:
We would probably want to skip smartphones here to avoid the delay there.

Reply:
Agreed.